parser/nodetype_string.go

```diff
@@ -0,0 +1,16 @@
+// generated by stringer -type=NodeType; DO NOT EDIT
```

Comment:
Yeah; you need to explicitly call `go generate` for it to be made, otherwise it'll be missing.